app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -208,11 +209,19 @@ class MainPageActivity : AppCompatActivity(), OnRequestPermissionsResultCallback
      * Starts the application guide.
      */
     private fun startAppGuide() {
-        if (!storage.getBoolOrDefault(GUIDE_KEY, false)) {
-            storage.setBoolean(GUIDE_KEY, true)
+        val signInStorage = storageOf(SIGN_IN)
+        if (!signInStorage.getBoolOrDefault(GUIDE_KEY, false)) {
+            signInStorage.setBoolean(GUIDE_KEY, true)
+
+            // Disable buttons so that the guide can be seen to completion
+            // TODO change buttonToActivity mapping to disable buttons
```

Comment:
The TODO could maybe be removed? (I'm not sure what our stance on disabling buttons during the guide was so Wiam will know more about this than I do)